server/src/test/java/org/elasticsearch/rest/action/info/RestClusterInfoActionTests.java

```diff
@@ -89,7 +89,8 @@ public void testHttpResponseMapper() {
                     .map(HttpStats::clientStats)
                     .map(Collection::stream)
                     .reduce(Stream.of(), Stream::concat)
-                    .toList()
+                    .toList(),
+                Map.of()
```

Comment:
Not related to the change itself, just surprised to see `.map(Collection::stream).reduce(Stream.of(), Stream::concat)`. Is there any benefit over `flatMap(Collection::stream)`?

Reply:
I don't think so. Sometimes a test can be deliberately written to avoid using the same pattern as the production code. But this does not seem to be the case either.